workspaces/semaphore/app-config.yaml

```diff
@@ -28,20 +28,25 @@ backend:
   # This is for local development only, it is not recommended to use this in production
   # The production database configuration is stored in app-config.production.yaml
   database:
-    client: better-sqlite3
-    connection: ':memory:'
+    client: pg
```

Comment:
To make sure this works for everyone you should probably use the in memory sqlite db rather than a full postgress one. 

When i ran this locally i used the in memory one and it worked fine.